### PR TITLE
Build deb package with omnibus

### LINKED ISSUE #1954: Missing CLI omnibus package for Ubuntu
We have `kontena-server` and `kontena-agent` Ubuntu packages, which act as systemd wrappers for running the Docker containers.

We could also have a `kontena-cli` package that contains some `/opt/kontena-cli_1.2.0` omnibus install of the CLI + ruby + deps.

### PR DESCRIPTION
**Pull request by jakolehm**
Binary deb package for Debian/Ubuntu.

Fixes #1954

### CODE REVIEW

File: cli/omnibus/package-scripts/kontena/postinst [hunk 1]
```diff
@@ -12,11 +12,7 @@ error_exit()
   exit 1
 }
 
-chown $USER /opt/kontena/embedded/bin
-chown $USER -R /opt/kontena/embedded/lib/ruby/gems/2.1.0
```
Comment (SpComb) [inline cli/omnibus/package-scripts/kontena/postinst]: I assume these are no longer needed after #1628?

Comment (jakolehm) [inline cli/omnibus/package-scripts/kontena/postinst]: Yes

File: cli/omnibus/package-scripts/kontena/postrm [hunk 2]
```diff
@@ -4,6 +4,6 @@
 # after package is uninstalled.
 #
 
-echo "Kontena has been uninstalled!"
+rm -f /usr/local/bin/kontena
```
Comment (SpComb) [inline cli/omnibus/package-scripts/kontena/postrm]: Maybe `[ -L /usr/local/bin/kontena ] && rm /usr/local/bin/kontena`

**Comment by SpComb** [comment]
Fixes #1954 (please add to description)

File: build/travis/omnibus_deb.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -e
+unset BUILD_ID
+cd cli/omnibus
+
+apt-get install -y -q libgecode-dev
```
Comment (SpComb) [inline build/travis/omnibus_deb.sh]: Does this need to run with `sudo`?

Comment (jakolehm) [inline build/travis/omnibus_deb.sh]: yes, fix coming..

**Pull request merged**